contracts/AcceleratingDistributor.sol

```diff
@@ -156,10 +156,10 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
         // payouts could eat into user balances. maxMultiplier is constrained to be at least 1e18 to enforce a minimum
         // 1x multiplier and avoid potential underflows.
         require(stakedToken != address(rewardToken), "Staked token is reward token");
+        require(maxMultiplier <= 1e24, "maxMultiplier too large"); // 1_000_000x multiplier.
         require(maxMultiplier >= 1e18, "maxMultiplier less than 1e18");
-        require(maxMultiplier < 1e36, "maxMultiplier can not be set too large");
-        require(secondsToMaxMultiplier > 0, "secondsToMaxMultiplier must be greater than 0");
-        require(baseEmissionRate < 1e27, "baseEmissionRate can not be set too large");
+        require(secondsToMaxMultiplier > 0, "secondsToMaxMultiplier <= 0");
```

Comment:
Minor nit: secondsToMaxMultiplier is unsigned, so it can only be 0 or positive. Consider "secondsToMaxMultiplier must be non-zero" ?